pages/search.js

```diff
@@ -105,6 +160,19 @@ class SearchPage extends React.Component {
               </FormGroup>
             </form>
           </Box>
+          <Box mt={4} mb={4} mx="auto">
+            <StyledFilters
+              filters={filters}
+              getLabel={key => intl.formatMessage(I18nFilters[key])}
```

Comment:
For the messages above to be translated properly

```suggestion
              getLabel={key => intl.formatMessage(I18nFilters[key], { count: 10 })}
```